src/app/migration/src/src/app/modules/workspace/services/editors/editor.service.ts

```diff
@@ -0,0 +1,61 @@
+import { ContentService } from '@sunbird/core';
+
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs/Observable';
+import { ConfigService, ServerResponse } from '@sunbird/shared';
+import { cleanSession } from 'selenium-webdriver/safari';
+
+
+/**
+ * Service to provides CRUD methods to make content api request by extending DataService.
+ *
+ */
+@Injectable()
+export class EditorService {
+    /**
+     * base Url for content api
+     */
+    baseUrl: string;
+    /**
+     * reference of config service.
+     */
+    public config: ConfigService;
+    /**
+     * reference of lerner service.
+     */
+    public contentService: ContentService;
+
+    public http: HttpClient;
+    public creator: string;
```

Comment:
create all of this as const inside function 
public creator: string;
public createdBy: string;
public createdFor: string
public organization: string;

Reply:
done